tools/src/main/java/org/apache/rocketmq/tools/command/consumer/ConsumerSubCommand.java

```diff
@@ -114,7 +114,7 @@ public void execute(CommandLine commandLine, Options options, RPCHook rpcHook) t
                             String result =
                                 ConsumerRunningInfo.analyzeProcessQueue(next.getKey(), next.getValue());
                             if (result.length() > 0) {
-                                System.out.printf(result);
+                                System.out.printf("%s", result);
```

Comment:
If no need format, just `System.out.println`?

Reply:
Yes,and this problem has been fixed.

Reply:
OK